TFT/src/User/config.ini

```diff
@@ -30,6 +30,7 @@
 # ensure that the following options are enabled in Marlin firmware.
 #
 # General options:
+#   G26_MESH_VALIDATION (in Configuration.h)
```

Comment:
PR for G26 is closed. is it really useful to put it in the config.ini?

Reply:
I think the author of the PR G26 is trying to reopen it without the problem he had.
However, yes it's better to remove the text from config.ini at the moment